src/main/scala/mesosphere/marathon/state/RunSpec.scala

```diff
@@ -12,6 +12,9 @@ import scala.collection.immutable.Seq
 
 //scalastyle:off
 trait RunSpec extends plugin.RunSpec {
+
+  type Id = PathId
```

Comment:
This doesn't quite work as you would expect, it will if and only if you put it in "object RunSpec".